backend/src/api/routes/s3-gateway/index.routes.ts

```
@@ -84,7 +85,33 @@ s3GatewayRouter.use((req, res, next) => {
+    next();
+    return;
+  }
+  const m = req.method.toUpperCase();
+  const p = req.path;
+  const q = new Set(Object.keys(req.query));
+  const hasKey = p.replace(/^\/+/, '').includes('/');
+  const isLargeBodyOp =
+    (m === 'PUT' && hasKey && !q.has('tagging') && !req.headers['x-amz-copy-source']) ||
+    (m === 'POST' && hasKey && q.has('uploadId'));
+  if (isLargeBodyOp) {
+    sendS3Error(res, 'EntityTooLarge', `Object exceeds max upload size (${appConfig.storage.maxS3UploadSize} bytes)`, {
```

ESLint

[error] Replace `res,·'EntityTooLarge',·`Object·exceeds·max·upload·size·(${appConfig.storage.maxS3UploadSize}·bytes)`,` with `⏎······res,⏎······'EntityTooLarge',⏎······`Object·exceeds·max·upload·size·(${appConfig.storage.maxS3UploadSize}·bytes)`,⏎·····`

(prettier/prettier)